third-party/rsutils/include/rsutils/version.h

```diff
@@ -5,6 +5,9 @@
 
 #include <string>
 #include <iosfwd>
+#if __cplusplus >= 201103L
```

Comment:
Can you explain?

Reply:
What I mean is, can't we just include `<cstdint>` always? Why the `#if`?

Reply:
This is due to an error in CI, but the problem has not yet been resolved.
I will test locally and try to fix it.